service/frontend/configs/quotas.go

```diff
@@ -66,6 +66,8 @@ var (
 		"PollWorkflowTaskQueue":              2,
 		"PollActivityTaskQueue":              2,
 		"GetWorkflowExecutionHistoryReverse": 2,
+		"GetWorkerBuildIdOrdering":           2,
+		"UpdateWorkerBuildIdOrdering":        2,
```

Comment:
🤷‍♂️. I figure they're roughly equally important as task polling, since you may really want to make sure that newly started workflows don't begin on some version you're unhappy with